cross/gnutls/Makefile

```diff
@@ -1,18 +1,19 @@
 PKG_NAME = gnutls
-PKG_VERS_SHORT = 3.4
-PKG_VERS = $(PKG_VERS_SHORT).17
+PKG_VERS_SHORT = 3.6
+PKG_VERS = $(PKG_VERS_SHORT).4
 PKG_EXT = tar.xz
 PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-PKG_DIST_SITE = ftp://ftp.gnutls.org/gcrypt/$(PKG_NAME)/v$(PKG_VERS_SHORT)
+PKG_DIST_SITE = https://www.gnupg.org/ftp/gcrypt/$(PKG_NAME)/v$(PKG_VERS_SHORT)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-DEPENDS = cross/libtasn1 cross/nettle cross/zlib
+DEPENDS = cross/libunistring cross/libtasn1 cross/nettle cross/zlib
 
 HOMEPAGE = http://www.gnu.org/software/gnutls/
 COMMENT  = GnuTLS is a secure communications library implementing the SSL, TLS and DTLS protocols and technologies around them
 LICENSE  = LGPL
 
 GNU_CONFIGURE = 1
 CONFIGURE_ARGS = --disable-doc --without-p11-kit --disable-nls gl_cv_func_gettimeofday_clobber=no gl_cv_func_signbit_gcc=no
+CONFIGURE_ARGS += --prefix=$(STAGING_INSTALL_PREFIX) --libdir=$(STAGING_INSTALL_PREFIX)/lib --includedir=$(STAGING_INSTALL_PREFIX)/include
```

Comment:
@ymartin59 This is what I was referring to. It should be `$(INSTALL_PREFIX)` with `libdir` and `includedir` removed, right?

Reply:
Done.